translations/monero-core_tr.ts

```diff
@@ -0,0 +1,2519 @@
+        <translation>&lt;a href=&apos;#&apos;&gt;Adres defterine&lt;/a&gt; yapıştırın veya seçim yapın</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="235"/>
+        <source>QR Code</source>
+        <translation>QR Kod</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="259"/>
+        <source>Resolve</source>
+        <translation>Çözme</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="277"/>
+        <source>No valid address found at this OpenAlias address</source>
+        <translation>Bu OpenAlias ​​adresinde geçerli bir adres bulunamadı</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="282"/>
+        <source>Address found, but the DNSSEC signatures could not be verified, so this address may be spoofed</source>
+        <translation>Adres bulundu, ancak DNSSEC imzaları doğrulanamadığından bu adres sahte olabilir</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="284"/>
+        <source>No valid address found at this OpenAlias address, but the DNSSEC signatures could not be verified, so this may be spoofed</source>
+        <translation>Bu OpenAlias ​​adresinde geçerli bir adres bulunamadı, ancak DNSSEC imzaları doğrulanamadığından bu sahte olabilir</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="287"/>
+        <location filename="../pages/Transfer.qml" line="290"/>
+        <source>Internal error</source>
+        <translation>İç hata</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="293"/>
+        <source>No address found</source>
+        <translation>Adres bulunamadı</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="312"/>
+        <source>Description &lt;font size=&apos;2&apos;&gt;( Optional )&lt;/font&gt;</source>
+        <translation>Açıklama &lt;font size=&apos;2&apos;&gt; (İsteğe Bağlı) &lt;/font&gt;</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="318"/>
+        <source>Saved to local wallet history</source>
+        <translation>Yerel cüzdan geçmişine kaydedildi</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="326"/>
+        <source>Send</source>
+        <translation>Gönderme</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="391"/>
+        <source>Show advanced options</source>
+        <translation>Gelişmiş ayarları göster</translation>
+    </message>
+    <message>
+        <location filename="../pages/Transfer.qml" line="454"/>
+        <source>Sweep Unmixable</
```

Comment:
don't know the structure here, but the result should be the translation of `from %1 to %2` as for `from A to B`